02/app.js

```diff
@@ -1,2 +1,15 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+
+ for (i =result; i<=x; i++){
```

Comment:
Uwaga, trzeba zawsze pamiętać o `let` tj. `for(let i ...` - jeśli tak nie napiszemy to tworzymy zmienną globalną co jest złą praktyką.